src/annotator/integrations/html-metadata.js

```diff
@@ -82,8 +60,9 @@ export class HTMLMetadata {
    * @return {string}
    */
   uri() {
+    const links = this._getLinks({ dc: {}, highwire: {} });
```

Comment:
This method is only interested in the last `link[rel=canonical]` link from the page, so the `dc` and `highwire` properties are not used.

Reply:
Should that be added as a code comment?